packages/request-client.js/src/http-data-access.ts

```diff
@@ -4,6 +4,9 @@ import axios, { AxiosRequestConfig } from 'axios';
 
 import { EventEmitter } from 'events';
 
+const packageJson = require('../package.json');
```

Comment:
you could do `import { version } from "../package.json"`

Reply:
You can't import a json file with `import` unless you declare it
```
declare module "*.json" {
  const value: any;
  export default value;
}
```

I personally prefer using require